### PR TITLE
Add Platform operator concept content

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Jira issue: https://issues.redhat.com/browse/AAP-516 - AAP Operator Installation guide

- Changed operator attribute values per updates in spreadsheet
- Added concept content

Preview here: http://file.emea.redhat.com/~ariordan/operator/operator-concept/build/tmp/en-US/html-single/

### CODE REVIEW

File: downstream/assemblies/platform/assembly-operator-install-planning.adoc [hunk 1]
```diff
@@ -10,9 +10,19 @@ ifdef::context[:parent-context: {context}]
...
+
+You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the individual scenarios to determine which meets your requirements.
+
+== About {OperatorPlatform}
+
+The {OperatorPlatform} provides cloud-native, push-button deployment of new {PlatformNameShort} instances in your OpenShift environment. 
+The {OperatorPlatform} includes resource types to deploy and manage instances of {ControllerNameStart} and Private Automation hub. It also includes {ControllerName} job resources for defining and launching jobs inside your {ControllerName} deployments. 
+You can install the {OperatorPlatform} from the Red Hat Operators catalog in OperatorHub.
+
+Deploying {PlatformNameShort} instances with a Kubernetes native operator offers advantages over launching instances from a playbook deployed on {OCP}. The {OperatorPlatform} supports upgrades and provides full lifecycle support for your {PlatformName} deployments.
```
Comment (TarikFD) [inline downstream/assemblies/platform/assembly-operator-install-planning.adoc:25]:
```suggestion
Deploying {PlatformNameShort} instances with a Kubernetes native operator offers several advantages over launching instances from a playbook deployed on {OCP}, including upgrades and full lifecycle support for your {PlatformName} deployments. 

You can install the {OperatorPlatform} from the Red Hat Operators catalog in OperatorHub.
```

File: downstream/assemblies/platform/assembly-operator-install-planning.adoc [hunk 2]
```diff
@@ -10,9 +10,19 @@ ifdef::context[:parent-context: {context}]
...
-You can use this section to help plan your {PlatformName} operator installation. Before installing, review the individual scenarios to determine which meets your requirements.
+OpenShift operators help install and automate day-2 operations of complex, distributed software on {OCP}. The {OperatorPlatform} enables you to deploy and manage {PlatformNameShort} components on {OCP}.
+
+You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the individual scenarios to determine which meets your requirements.
+
+== About {OperatorPlatform}
+
+The {OperatorPlatform} provides cloud-native, push-button deployment of new {PlatformNameShort} instances in your OpenShift environment. 
+The {OperatorPlatform} includes resource types to deploy and manage instances of {ControllerNameStart} and Private Automation hub. It also includes {ControllerName} job resources for defining and launching jobs inside your {ControllerName} deployments. 
+You can install the {OperatorPlatform} from the Red Hat Operators catalog in OperatorHub.
```
Comment (TarikFD) [inline downstream/assemblies/platform/assembly-operator-install-planning.adoc:23]:
```suggestion
```

Comment (TarikFD) [reply downstream/assemblies/platform/assembly-operator-install-planning.adoc:23]: (Moving the "action item" from line 23 down to the end)

File: downstream/assemblies/platform/assembly-operator-install-planning.adoc [hunk 3]
```diff
@@ -10,9 +10,19 @@ ifdef::context[:parent-context: {context}]
 :context: operator-install-planning
 
 [role="_abstract"]
-OpenShift operators help install and automate day-2 operations of complex, distributed software on OpenShift.
+{PlatformName} is supported on both Red Hat Enterprise Linux 8 and Red Hat Openshift.
 
-You can use this section to help plan your {PlatformName} operator installation. Before installing, review the individual scenarios to determine which meets your requirements.
+OpenShift operators help install and automate day-2 operations of complex, distributed software on {OCP}. The {OperatorPlatform} enables you to deploy and manage {PlatformNameShort} components on {OCP}.
+
+You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the individual scenarios to determine which meets your requirements.
```
Comment (TarikFD) [inline downstream/assemblies/platform/assembly-operator-install-planning.adoc:17]:
```suggestion
You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.
```

**Review by TarikFD — COMMENTED** [review]
@ariordan-redhat - some minor changes for you to look at

File: downstream/assemblies/platform/assembly-operator-install-planning.adoc [hunk 4]
```diff
@@ -10,9 +10,20 @@ ifdef::context[:parent-context: {context}]
 :context: operator-install-planning
 
 [role="_abstract"]
-OpenShift operators help install and automate day-2 operations of complex, distributed software on OpenShift.
+{PlatformName} is supported on both Red Hat Enterprise Linux 8 and Red Hat Openshift.
 
-You can use this section to help plan your {PlatformName} operator installation. Before installing, review the individual scenarios to determine which meets your requirements.
+OpenShift operators help install and automate day-2 operations of complex, distributed software on {OCP}. The {OperatorPlatform} enables you to deploy and manage {PlatformNameShort} components on {OCP}.
+
+You can use this section to help plan {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-operator-install-planning.adoc:17]:
```suggestion
You can use this section to help plan your {PlatformName} installation on your {OCP} environment. Before installing, review the supported installation scenarios to determine which meets your requirements.
```

**Comment by TarikFD** [comment]
Looks good @ariordan-redhat . Merging into the operator feature branch

**Pull request merged**